cardano-cli/test/cardano-cli-golden/files/golden/alonzo/transaction-view.out

```diff
@@ -9,6 +9,7 @@ inputs:
 metadata: null
 mint: null
 outputs: []
+redeemers: []
```

Comment:
Can you add a test case for non-empty redeemers?

Reply:
Added in [this commit](https://github.com/IntersectMBO/cardano-cli/pull/664/commits/c911b1963ce35cc59d1b2497cf9168207d8b6d91) :+1: